### DISCUSSION TITLE
Create method body, even if there is one

### DISCUSSION THREAD

**Discussion by benehmb** [community/community#7458] [Copilot Conversations]
If a method already has a body, copilot will ignore it and add one as you can see in this screenshot below:

![image](https://user-images.githubusercontent.com/51318117/140715631-0db413c4-6011-4229-a7a9-a2910d7f4cfc.png)

I'm not sure if this is is a bug or possible to fix. I just wanted to give some feedback.

**Comment by jansorg** [comment]
Thanks! Which editor were you using?

**Reply by benehmb** [reply]
I am using Rider
